tests/config/meilisearch.yaml

```diff
@@ -40,3 +40,18 @@ meilisearch:
             class: 'Meilisearch\Bundle\Tests\Entity\DummyCustomGroups'
             enable_serializer_groups: true
             serializer_groups: ['public', 'private']
+        -   name: dynamic_settings
+            class: 'Meilisearch\Bundle\Tests\Entity\DynamicSettings'
+            settings:
+                filterableAttributes:
+                    _service: '@Meilisearch\Bundle\Tests\Integration\Fixtures\FilterableAttributes'
```

Comment:
Why did you have to use the `@Meilisearch` notation here, and we don't need it in the `class`? Is it a design definition?

Reply:
that's how services are referenced in the Symfony config